documentation/chains.md

```diff
@@ -373,16 +373,15 @@ const entryObj = await factomConnectSDK.chains.entries.create({
-| `params.content`          | required                         | string</br> This is the data that will be stored directly</br>  on the blockchain. Please be sure that no</br>  private information is entered here.</br>   **Note:** The value in `content` parameter will</br>  be encoded in Base64 format by Connect</br>  SDK.                                                                                                                                                                                                                                                                                                                                                        | **content is required.**</br>   `content` parameter was not provided.</br>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
-| `params.signerChainId`    | required</br> or</br> optional   | string</br> The chain ID of the signer identity.</br>  **Note:** This parameter is optional for creating an unsigned entry. However, if `signerPrivateKey` is inputted then `signerChainId` must also be inputted.                                                                                                                                                                                                                                                                                                                                                                                       | In case of creating a signed entry:</br> **signerChainId is required.**</br> `signerChainId` parameter was not provided.</br></br>  In case of creating an unsigned entry:</br>  **signerChainId is required when passing a signerPrivateKey.**</br> `signerPrivateKey` was provided but lacking `signerChainId` parameter.                                                                                                                                                                                                                                                                                                                                                                                                   |

```

Comment:
It appears you deleted the `params.signerChainId` row here. Just for clarification, was that intentional?